src/subscription/SubscriptionPage.messages.js

```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  'subscription.heading.page': {
+    id: 'subscription.heading.page',
+    defaultMessage: 'Subscription',
+    description: 'Page heading for subscription.',
+  },
+  'subscription.loading.details': {
+    id: 'subscription.loading.details',
+    defaultMessage: 'Loading subscription...',
+    description: 'Message when subscription page is being loaded',
+  },
+  'subscription.loading.error': {
```

Comment:
I don't see this being used? Is it in another commit?

Reply:
Removed